Makefile

```diff
@@ -104,8 +104,7 @@ tar: .podman-from-container
 	cp -r conf/containers $(ASSET_DIR)/etc/containers
 	cp -r conf/cni $(ASSET_DIR)/etc/cni
 	cp README.md $(ASSET_DIR)/
-	cp -r $(IMAGE_ROOTFS)/usr/local/lib $(ASSET_DIR)/usr/local/lib
-	cp -r $(IMAGE_ROOTFS)/usr/local/bin $(ASSET_DIR)/usr/local/bin
+	$(DOCKER) run --rm $(PODMAN_IMAGE) tar c /usr/local/{bin,lib} | tar -xC $(ASSET_DIR)
```

Comment:
What's the purpose of this change?

Reply:
Oh sorry, I pushed more commits for this PR than I intended.
I don't *fully* understand how the Makefile is exporting the images, but when I try it on my Mac, it doesn't behave as expected and I couldn't pull the contents from those dirs. I don't recall what the error was at this stage.

In any case, it's not a change that needs to be upstreamed; I'll pick out this commit when I get a chance.

Reply:
Oh! I *think* the problem is that `.podman-from-container` would need to depend on `multiarch-images` for this to work, and because I wasn't *running* `multiarch-images`, which I didn't need, the images weren't being exported, thus the files being copied did not exist.

Does that sound like a viable explanation?

Reply:
I see. The purpose of this Makefile target is only to extract the binaries from the image in order to publish them as tar archive that is attached to each GitHub release. Thus, so far it was always run after the `multiarch-images` target in practice.